deploy/telemeter-server.yaml

```diff
@@ -104,3 +107,8 @@ items:
           - --tls-key=/etc/pki/service/tls.key
           - --tls-crt=/etc/pki/service/tls.crt
           - --authorize=http://telemeter-authorization
+parameters:
+- name: IMAGE
+  value: openshift/origin-telemeter
+- name: IMAGE_TAG
+  value: v3.11
```

Comment:
post-mortem comment: this should be `v4.0`, no?

Reply:
I think we should specify that in the saasherder config https://github.com/app-sre/saas-telemeter/blob/master/telemeter-services/telemeter-server.yaml#L10.

@riuvshin kept it this way to be identical to the original manifest. 

We can make a follow up PR and update the image to quay.io/openshift/origin-telemeter and the tag to v4.0.0 but it's independent of this change

Reply:
exactly, thanks @squat :)